client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/ManualContactsForm/ManualContactsForm.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import {Grid, IconButton, Typography} from '@material-ui/core';
+import {AddCircle as AddCircleIcon} from '@material-ui/icons';
+import ContactForm from './ContactForm/ContactForm';
+import {defaultContact} from '../../InteractionSection/InteractionEventForm';
+import {useFieldArray, useFormContext} from 'react-hook-form';
+import Contact from 'models/Contact';
+import InteractionEventDialogFields
+    from 'models/enums/InteractionsEventDialogContext/InteractionEventDialogFields';
+import useFormStyles from 'styles/formStyles';
+import useStyles from '../../InteractionSection/InteractionEventFormStyles';
+
+const addContactButton: string = 'הוסף מגע';
+
+const ManualContactsForm = () => {
+    const { control } = useFormContext();
+    const { fields, append } = useFieldArray<Contact>({ control, name: InteractionEventDialogFields.CONTACTS });
+    const contacts = fields;
+
+    const formClasses = useFormStyles();
+    const classes = useStyles();
+
+    return (
+        <Grid container>
+            <div className={classes.newContactFieldsContainer}>
+                {
+                    contacts.map((contact, index: number) => (
+                        <Grid key={index} item className={classes.contactedPersonContainer}>
+                            <ContactForm
+                                key={index}
+                                updatedContactIndex={index}
+                                contactStatus={contact.contactStatus}
+                                contactCreationTime={contact.creationTime}
+                            />
+                        </Grid>
+                    ))
+                }
+                <Grid item>
+                    <IconButton
+                        test-id='addContact'
+                        onClick={() => append(defaultContact)}
+                    >
+                        <AddCircleIcon color='primary'/>
+                    </IconButton>
+                    <Typography
+                        variant='caption'
+                        className={formClasses.fieldName + ' ' + classes.fieldNameNoWrap}
+                    >
+                        {addContactButton}
```

Comment:
![image](https://user-images.githubusercontent.com/24894463/100364167-6ce7ce00-3006-11eb-95c3-b8442f788723.png)
The fields collide with each other, ןn addition the width can be increased

Reply:
the fields looks good know, remove the unnecessary scroll 
![image](https://user-images.githubusercontent.com/24894463/100542626-c8f36200-3253-11eb-921d-59d64c51b1c1.png)